.github/workflows/tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+
+jobs:
+  test:
+    name: "Go ${{ matrix.go-version }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go-version: [1.14.x, 1.15.x, 1.16.x]
```

Comment:
The project requires Golang 1.16 or greater (defined in `go.mod`; see also README). However, it does not utilize any new features that are not also available in 1.14 or greater.
So, if you want to support development with older versions, you could change that in `go.mod`, but you would need to update documentation and Dockerfile to use `GO111MODULE=on` environment variable for `go get/build/run` commands.
IMO, it makes more sense to require 1.16, though.

Reply:
All right, your wish is my command ;].